documentation/assemblies/assembly-iot-guide.adoc

```diff
@@ -0,0 +1,20 @@
+// This assembly is included in the following assemblies:
+//
+// master-openshift.adoc
+
+:context: {context}-iot
+
+[id='iot-guide-{context}]
+= IoT guide
+
+The IoT guide provides resources on how to set up and manage {ProductName} IoT features.
+
+include::../modules/proc-installing-using-bundle.adoc[leveloffset=+1]
+
+include::../modules/proc-installing-iot-services.adoc[leveloffset=+1]
+
+include::../iot/common/proc-creating-iot-project.adoc[leveloffset=+1]
```

Comment:
These should all be directly under modules (no hierarchy) if i've understood @ncbaratta and @jenmalloy correctly

Reply:
Yeah, I started like that, but figured we'll add a lot of clutter as we continue adding docs in this area.

Reply:
I agree @dejanb but the idea is to let anyone use our modules and so all projects have to stick to the same model of putting them all together ... and being cluttered :(

Reply:
@dejanb, one way to make clear which modules are specific to IoT might be to use a naming scheme like this: proc-iot-file-name.adoc
@ncbaratta, does this suggestion violate any of our rules?

Reply:
That is a-ok!